src/Rings/AbelianClosure.jl

```diff
@@ -526,6 +526,41 @@ end
 (R::QQField)(a::QQAbFieldElem) = R(a.data)
 (R::ZZRing)(a::QQAbFieldElem) = R(a.data)
 
+################################################################################
+#
+#  Conversion to `QQBarFieldElem`
+#
+#  We assume the natural embedding of cyclotomic fields into `QQBarField()`
+#  that is given by the (documented) fact that
+#  `root_of_unity(F::QQBarField, n::Int)` is $\exp(2 \pi i / n)$.
+#
+################################################################################
+
+function (F::QQBarField)(a::QQAbFieldElem)
+  N = a.c
+  cfs = Oscar.coefficients(a.data)
+  r = root_of_unity(F, N)
+  pow = one(F)
+  res = cfs[1] * pow
+  for i in 2:length(cfs)
+    pow = pow * r
+    res = res + cfs[i] * pow
+  end
+  return res
+end
+
+Nemo.QQBarFieldElem(a::QQAbFieldElem) = algebraic_closure(QQ)(a)
```

Comment:
Not sure we need this. Ideally, everyone should use parents :)

Reply:
My idea was to provide a constructor analogous to the list in `Nemo.jl/src/calcium/qqbar.jl`.
Bad idea?

Reply:
Hm, true, might be useful for internal use to avoid the `algebraic_closure(QQ)(...)`.